app/code/Magento/Directory/Block/Data.php

```diff
@@ -196,7 +197,49 @@ public function getRegionHtmlSelect()
         )->setClass(
             'required-entry validate-state'
         )->setValue(
-            (int)$this->getRegionId()
+            intval($this->getRegionId())
+        )->setOptions(
+            $options
+        )->getHtml();
+        \Magento\Framework\Profiler::start('TEST: ' . __METHOD__, ['group' => 'TEST', 'method' => __METHOD__]);
+        return $html;
+    }
+
+    /**
+     * Returns region html select
+     *
+     * @param null|string $value
+     * @param string $name
+     * @param string $id
+     * @param string $title
+     * @return string
+     */
+    public function getRegionSelect($value = null, $name = 'region', $id = 'state', $title = 'State/Province')
```

Comment:
Could you please add typehints for method arguments and return typehint ?

Reply:
Is adding `declare(strict_types=1); ` at the beginning necessary ?

Reply:
According to [Technical Guidelines](https://devdocs.magento.com/guides/v2.3/coding-standards/technical-guidelines.html#1-basic-programming-principles) `#1.3.1`, this is *should* be done.
But as we are introducing a new method, it is better to use strict types, as is described in `#1.3` and where PHP is aiming ([PSR-12](https://github.com/php-fig/fig-standards/blob/master/proposed/extended-coding-style-guide.md)).

Reply:
I think this is better to use `?int $value = null,` instead of `string $value = null,` to avoid double conversion to int.

Reply:
yeah, sure.